src/main/java/com/kaziamyr/onlinebookstore/controller/BookController.java

```diff
@@ -24,12 +30,29 @@ public List<BookDto> findAll() {
     }
 
     @GetMapping("/{id}")
-    public BookDto getBookById(@PathVariable Long id) {
+    public BookDto getById(@PathVariable Long id) {
         return bookService.getBookById(id);
     }
 
     @PostMapping
     public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
     }
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @DeleteMapping("/{id}")
+    public void deleteById(@PathVariable Long id) {
+        bookService.deleteById(id);
+    }
+
+    @PutMapping("/{id}")
+    public BookDto updateById(@PathVariable Long id, @RequestBody CreateBookRequestDto bookDto) {
```

Comment:
could be just `update`
```suggestion
    public BookDto update(@PathVariable Long id, @RequestBody CreateBookRequestDto bookDto) {
```